---
layout: default
title: Etai Jacob
---
<div class="blurb">
	<br><br>
	<h3>Source code:</h3> 
	<h2>Codon-level information improves predictions of inter-residue contacts in proteins by correlated mutation analysis.</h2>
	<em>eLife 2015;10.7554/eLife.08932 </em>
	<br><br>
	<p><a href="http://dx.doi.org/10.7554/eLife.08932">link to manuscript (eLIFE website)</a></p>
	<p><a href="https://github.com/etaijacob/MAPDB">R source code for structure-domain sequence mapping</a></p>
	<p><a href="https://github.com/etaijacob/AA2CODON">A python package for generating codon MSAs</a></p>
	<p><a href="https://github.com/etaijacob/CMA">An R package for correlated mutation analysis both at codon and amino acid levels</a></p>
	<p><a href="https://github.com/etaijacob/ACPSICOV">Protein Sparse Inverse COVariance analysis program for nucleotide and amino acid sequences</a></p>
	
</div><!-- /.blurb -->
