---
layout: default
title: Etai Jacob
---
<div class="blurb">
	<br><br>
	<h2>About me</h2>
	<h1>I obtained my PhD in computational biology in 2016 from Bar-Ilan University and the Weizmann Institute of Science (as a collaboration) in Israel. I worked at Compugen Ltd. (www.cgen.com) from 2006 to 2016 as a scientist, developing data analysis methods and algorithms for drug discovery, using mostly machine learning and statistics. I am currently a computational biologist/data scientist at the department of data sciences at Dana Farber Cancer Institute and the Broad Institute of MIT and Harvard. 
Other facts about me: My Erdos # is 3 (Erdos to Peter Winkler to Ron Unger to me). I started as an embedded software engineer during my bachelorâ€™s degree studies (Computer Science) and moved to computational biology a few years after.<h3>
	<br><br>
	<h3>Source code:</h3> 
	<h3>Codon-level information improves predictions of inter-residue contacts in proteins by correlated mutation analysis.</h3>
	<em>eLife 2015;10.7554/eLife.08932 </em>
	<br><br>
	<p><a href="http://dx.doi.org/10.7554/eLife.08932">link to manuscript (eLIFE website)</a></p>
	<p><a href="https://github.com/etaijacob/MAPDB">R source code for structure-domain sequence mapping</a></p>
	<p><a href="https://github.com/etaijacob/AA2CODON">A python package for generating codon MSAs</a></p>
	<p><a href="https://github.com/etaijacob/CMA">An R package for correlated mutation analysis both at codon and amino acid levels</a></p>
	<p><a href="https://github.com/etaijacob/ACPSICOV">Protein Sparse Inverse COVariance analysis program for nucleotide and amino acid sequences</a></p>
	
</div><!-- /.blurb -->
